helpers/constants.py

```diff
@@ -55,6 +55,7 @@
 ZAP_JSON_REPORT_OPTION = "-J"
 ZAP_OTHER_OPTIONS = "-z"
 ZAP_HOOK_OPTION = "--hook"
+ZAP_ACTIVE_SCAN_POLICY_NAME = "Default Policy"
```

Comment:
zap when performing a fullscan creates a policy called "Default Policy" it's needed to specify that name in order to change the scan rules in it.

Reply:
add a // NOTE: for that one if you can